src/components/listitems/ListItemTransactionRadio.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from "react";
+import { View } from "react-native";
+import RNReactNativeHapticFeedback from "react-native-haptic-feedback";
+import { IOStyles, useIOExperimentalDesign } from "src/core";
+import { Icon } from "../icons";
+import { AnimatedRadio } from "../radio/AnimatedRadio";
+import { HSpacer, VSpacer } from "../spacer";
+import { H6 } from "../typography";
+import { LabelSmall } from "../typography/LabelSmall";
+import { PressableListItemBase } from "./PressableListItemsBase";
+
+type ListItemTransactionRadioProps = {
+  selected?: boolean;
+  amountString: number;
+} & (
+  | {
+      isSuggested?: false;
+    }
+  | {
+      isSuggested: true;
+      suggestReason: string;
+    }
+);
+export const ListItemTransactionRadio = (
+  props: ListItemTransactionRadioProps
+) => {
+  const [toggleValue, setToggleValue] = React.useState(props.selected ?? false);
+  const pressHandler = () => {
+    RNReactNativeHapticFeedback.trigger("impactLight");
+    setToggleValue(val => !val);
+  };
+  const isExperimental = useIOExperimentalDesign();
+  return (
+    <PressableListItemBase onPress={pressHandler}>
+      <View>
+        <LabelSmall color="black" weight="SemiBold">
+          Label
```

Comment:
```suggestion
          Label
```
Hardcoded label? 🤔

Reply:
whoopsie, that's a hard miss 😅 
addressed in [86ab979](https://github.com/pagopa/io-app-design-system/pull/139/commits/86ab9790177fe4ed7ef2c9560635c01f03991d49)